licenses/tests/test_models.py

```diff
@@ -2,13 +2,13 @@
 from django.utils.translation import override
 
 from i18n import DEFAULT_LANGUAGE_CODE
-from licenses import FREEDOM_LEVEL_MIN, FREEDOM_LEVEL_MID, FREEDOM_LEVEL_MAX
-from licenses.models import (
-    LegalCode,
-    License,
-    LicenseLogo,
-    TranslatedLicenseName,
-)
+from licenses import FREEDOM_LEVEL_MAX
```

Comment:
Can we combine these into two import statements?

Reply:
done